### PR TITLE
Feature/midl demo requests

### PR DESCRIPTION
**Pull request by pmandler**
Add time to submission due. 
Add word 'optional' to paper removal (given that the code of conduct isn't optional).
Reviewers cannot reveal name in reviews.
Add option for oral presentation to review form.
Increase size of review to be essentially unlimited.

### CODE REVIEW

File: venues/MIDL.io/2019/Conference/python/init_conference.py [hunk 1]
```diff
@@ -34,7 +34,7 @@
         },
         "remove if rejected": {
             "order": 12,
-            "value-checkbox": "Remove submission if paper is rejected.",
+            "value-checkbox": "(optional) Remove submission if paper is rejected.",
```
Comment (melisabok) [inline venues/MIDL.io/2019/Conference/python/init_conference.py]: the ui shoes the required field, is this still needed?

**Comment by melisabok** [comment]
@pmandler should I merge this? I guess this is in live already.

File: venues/MIDL.io/2019/Conference/python/init_conference.py (hunk 1 above)
Comment (pmandler) [reply venues/MIDL.io/2019/Conference/python/init_conference.py]: This was a specific request resulting from the demo last Friday.

**Comment by pmandler** [comment]
Yes it is.  It was one of the requests resulting from the demo last Friday.

**Comment by melisabok** [comment]
This code is in live, why did you close it?

**Comment by pmandler** [comment]
Accidentally.  I meant to comment, not close.

**Pull request merged**